javaBox.js

```diff
@@ -75,15 +76,40 @@ function dockerCommand(command, commandTimeLimit, nextCommand) {
     const execution = (err, container) => {
 
         if (!err) container.exec(opts, (err, exec) => {
-            exec.start((err, stream) => waitCmdExit(container, exec, nextCommand, stream, commandTimeLimit));
+            exec.start((err, stream) => {
```

Comment:
You should be able to obtain the complete log from the container at the end of its execution, before deleting it. But this solution is fine as well.